subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1295,9 +1295,10 @@ static int cmd_wifi_ap_enable(const struct shell *sh, size_t argc,
 			      char *argv[])
 {
 	struct net_if *iface = net_if_get_wifi_sap();
-	static struct wifi_connect_req_params cnx_params;
+	struct wifi_connect_req_params cnx_params;
```

Comment:
This patch change the lifetime of `cnx_params`. Can you confirm you have checked address of this variable is never copied somewhere else?

Reply:
With `"static"`, this parameter will carry configurations from last time. For example, I first enable AP of WPA-PSK mode and set psk in cnx_params. Later, I disabled this AP and start it as open mode. There will be warning log `"Passphrase provided without security configuration"`. 
I think it is meaningless to use "static" because AP configurations might be different for each time. 
This parameter only used in this API so I simply removed the keywork "static" and do memset of this parameter.